zproject/local_settings_template.py

```diff
@@ -298,3 +298,6 @@
 }
 
 CAMO_URI = ''
+
+# Set the rabbitmq host(s)
```

Comment:
Would probably expand this comment a bit -- "By default, Zulip connects to rabbitmq running locally on the machine, but Zulip also supports connecting to RabbitMQ over the network; to use a remote RabbitMQ instance, set RABBITMQ_HOST here."